.github/workflows/asf-updates.yml

```diff
@@ -25,6 +25,9 @@ jobs:
         with:
           python-version: '3.11'
 
+      - name: Install release helper dependencies
+        run: pip install --upgrade setuptools setuptools_scm
+
```

Comment:
nit: This actually wouldn't be necessary (we only use `set-dep-ver`, which doesn't need `setuptools_scm`), but I guess it's the safe thing to do, so we could leave it like this 😉